src/ray/common/ray_config_def.h

```diff
@@ -903,3 +903,6 @@ RAY_CONFIG(int, object_manager_client_connection_num, 4)
 //     std::min(std::max(2, num_cpus / 4), 8)
 // Update this to overwrite it.
 RAY_CONFIG(int, object_manager_rpc_threads_num, 0)
+
+// Write export API events to file if enabled
+RAY_CONFIG(bool, enable_export_api_write, false)
```

Comment:
it is not used anywhere?

Reply:
Not yet, but it will be used when we write events for each of the resource types. Pulled this out into a separate PR so we can work on the various resource in parallel